config/deploy.rb

```diff
@@ -39,6 +39,8 @@
 # We want Honeybadger to report deployments to our Capistrano stage names (e.g., dev, stage, prod)
 set :honeybadger_env, fetch(:stage)
 
+set :whenever_roles, %i[app courses]
```

Comment:
In other projects we've used a `cron` role for where the cron jobs run.  Can we do that here for consistency?

Reply:
See https://github.com/sul-dlss/purl-fetcher/blob/2cda3653ec435817e48f2f2bdbfbdee01ad6692c/config/deploy.rb#L26

Reply:
For SearchWorks we have cases where we want to run jobs on all the VMs and other cases where we want to run jobs on only one of the VMs per load balanced cluster. I'm not sure `cron` is clearer in this case. I agree there's opportunity for cleanup and clarifying names. If it's okay with you I'd prefer to fix the immediate issue with the existing roles and then work on some of the cleanup.

Reply:
The more involved cleanup could look like this https://github.com/sul-dlss/SearchWorks/pull/4311

Open to suggestions for better terms for the :db and :app roles.